pkg/build/builder/cmd/scmauth/password.go

```diff
@@ -72,12 +72,12 @@ func (u UsernamePassword) Setup(baseDir string) error {
 
```

Comment:
That's what I would expect. It's weird to get a default username out of the blue... that could even break servers which allow anonymous auth, but not incorrect random usernames